build.gradle

```diff
@@ -117,6 +134,8 @@ subprojects {
                 prometheus_client_common: "io.prometheus:simpleclient_common:${prometheusVersion}",
                 protobuf                : "com.google.protobuf:protobuf-java:${protobufVersion}",
                 protobuf_util           : "com.google.protobuf:protobuf-java-util:${protobufVersion}",
+                zipkin_reporter         : "io.zipkin.reporter2:zipkin-reporter:${zipkinReporterVersion}",
```

Comment:
you can switch to the bom https://github.com/openzipkin/zipkin-reporter-java#version-alignments

gradle supports it now, then you also get zipkin-junit's version number for free.

Reply:
cool. I think this will require a bit of rework of our gradle files, so I'll log this as a separate issue, so as not to clutter up this PR too much.

Reply:
#1111